frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -270,7 +269,7 @@ function BookInfo.getDocProps(file, book_props, no_open_document, no_customize)
         end
     end
 
-    return no_customize and (book_props or {}) or BookInfo.customizeProps(book_props, file)
+    return no_customize and (book_props or {}) or BookInfo.extendProps(book_props, file)
```

Comment:
Guess `no_customize` could be renamed to `not_extended` or `plain` or `raw` or something else like that.